### ISSUE TITLE
Embedded pulse images fail in some email clients (Thunderbird, iOS and OS X Mail.app)

### ISSUE THREAD

**Issue by maliayas** [metabase/metabase#1954]
Thunderbird has a bug with embedded images. It can't show embedded images in some cases depending on the `Content-Type` header. You can read about [here](https://bugzilla.mozilla.org/show_bug.cgi?id=488987) and [here](https://bugzilla.mozilla.org/show_bug.cgi?id=61815).

In my tests, when I replaced `Content-Type: multipart/mixed` with `Content-Type: multipart/related` it worked. Don't know if this change breaks other clients.

PS: There is a typo in pulse emails. In one of the inline CSS, it reads `:font-family` instead of `font-family:`

Thanks.

:arrow_down: Please click the :+1: reaction instead of leaving a `+1` or `update?` comment

**Comment by maliayas** [comment]
Here is a screenshot. Image doesn't display. Tested on 2 different Thunderbird one on Windows 7 and other on Windows 8. Images are displayed as attachment at the bottom of the email (feature of Thunderbird).

![ekran goruntusu 2016 02 17 00_44_08](https://cloud.githubusercontent.com/assets/844008/13094576/382e13ba-d514-11e5-9097-671cfc20c7a7.png)

**Comment by maliayas** [comment]
Sorry for bugging, but do you think this issue might be fixed in the next release? Pulse emails are not readable now in Thunderbird due to this bug.

**Comment by StaymanHou** [comment]
Any update on this issue? I ran into this one, too.

**Comment by StaymanHou** [comment]
Any update on this issue? I ran into this one, too.

**Comment by camsaul** [comment]
It sounds like this is a bug with Thunderbird, not Metabase, right? Although I'm not sure at this rate it's going to get fixed any time soon, it looks like [it's been open 17 years](https://bugzilla.mozilla.org/show_bug.cgi?id=61815) 😳

**Comment by maliayas** [comment]
Yep, that's the case. However considering how popular ThunderBird is, 
would it be an option to implement the workaround I mentioned in the OP?

Thanks for taking time on this issue.

Ali Ayas

2017-11-11 04:52 tarihinde Cam Saul yazdı:

**Comment by etlam** [comment]
Any update on this issue? I ran into this one, too.

**Comment by cdeweyx** [comment]
Unable to repro this in v53, closing for now and we'll re-open if this surfaces again. 🙏
